interp/runner.go

```diff
@@ -916,5 +916,9 @@ func (r *Runner) open(ctx context.Context, path string, flags int, mode os.FileM
 }
 
 func (r *Runner) stat(name string) (os.FileInfo, error) {
```

Comment:
curious if you are thinking these would never benefit from a ctx param or not (thinking log correlation)

Reply:
Good catch @codefromthecrypt; StatHandlerFunc should take a context, even if just for consistency with the other handler funcs.

Reply:
Agreed. Now stat handler contains a context.